test/chai-extensions.js

```diff
@@ -0,0 +1,145 @@
+var chai = require('chai');
+var utils = chai.utils;
+
+chai.use(function(chai, utils) {
+    /**
+     * Error assertion for `Errors` instances.
+     */
+    chai.Assertion.addChainableMethod('error', function(message) {
```

Comment:
Would document those extensions at the wiki page, otherwise this could get confusing